services/ProductRegistrationService.php

```diff
@@ -88,6 +88,8 @@ private function optInStrategy($email) {
         curl_setopt($curl, CURLOPT_POST, true);
         curl_setopt($curl, CURLOPT_POSTFIELDS, http_build_query(array('email' => $email)));
         curl_setopt($curl, CURLOPT_RETURNTRANSFER, true);
+        curl_setopt($curl, CURLOPT_SSL_VERIFYPEER, false);
```

Comment:
@MatthewVita @sjpadgett 
Isn't this odd that this is required for Windows and not Linux? (this is supposed to default to true on all OS's)
The ssl certificate is served by a load balancer on aws and I couldn't find any documented issues online regarding this type of OS specific behavior.
btw @sjpadgett , thanks for pinpointing this issue!